setup.py

```diff
@@ -17,15 +17,15 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        'requests>=2.4.2,<2.29',
-        'future>=0.16,<0.19',
-        'python-magic>=0.4,<0.5',
+        'requests>=2.4.2',
```

Comment:
There's a 3.0 branch on the requests repo but no indication how soon it's coming or if that'll be a significant interface change. Could be worth including `<3.0`, but up to you.

Reply:
It looks like the requests 3.0 branch hasn't been touched since 2018ish, and I'm not sure whether any breaking changes would necessarily affect the Client or not even if/when released.  My proposal: until it is demonstrated that some aspect of our Client is breaking with a newer version (whether thats 2.x or 3.x), I don't think it is worth the incompatibility for users wanting to download bleeding edge versions.